visualization/app/codeCharta/ui/codeMap/codeMap.render.service.ts

```diff
@@ -33,13 +35,21 @@ export function getMapResolutionScaleFactor(files: FileState[]) {
 	}
 }
 
-export class CodeMapRenderService {
+export class CodeMapRenderService implements IsLoadingFileSubscriber{
 	constructor(
+		private $rootScope: IRootScopeService,
 		private storeService: StoreService,
 		private threeSceneService: ThreeSceneService,
 		private codeMapLabelService: CodeMapLabelService,
 		private codeMapArrowService: CodeMapArrowService
-	) {}
+	) {
+		IsLoadingFileService.subscribe(this.$rootScope, this)
+	}
+	onIsLoadingFileChanged(isLoadingFile: boolean) {
+		if (isLoadingFile && this.threeSceneService!==undefined) {
```

Comment:
I think you should run the prettier once :)

Reply:
I tried it yesterday, It just changes all the files and I land up with 500+ modified files. I might be doing something wrong